broadcast.php

```diff
@@ -17,10 +17,20 @@
 // URL parameters
 $action = isset($_REQUEST['action']) ? $_REQUEST['action'] : '';
 $text = isset($_REQUEST['text']) ? trim($_REQUEST['text']) : '';
+$tag_boxes = isset($_REQUEST['tag_boxes']) ? $_REQUEST['tag_boxes'] : array();
```

Comment:
This variable is a map of tag numbers to booleans indicating whether to include them or not, is that correct? I'd recommend a variable name that makes that clearer, like $include_flags_for_tags or something that suits you better.

Reply:
Updated it to $include_tags